docs/integration_fetching_data.md

```diff
@@ -70,41 +78,34 @@ async def async_setup_entry(hass, entry, async_add_entities):
     # Fetch initial data so we have data when entities subscribe
     await coordinator.async_refresh()
 
-    async_add_entities(MyEntity(coordinator, idx) for idx, ent
-                       in enumerate(coordinator.data))
+    async_add_entities(
+        MyEntity(coordinator, idx) for idx, ent in enumerate(coordinator.data)
+    )
+
+
+class MyEntity(CoordinatorEntity):
```

Comment:
This example should inherit `LightEntity` too to be correct.

Reply:
Good call, will adjust.